### ISSUE TITLE
Shortened URLs for lf.Exception explanation don't load.

### ISSUE THREAD

**Issue by freshp86** [google/lovefield#139]
Example http://sn.im/2a0j3wn?c=201
I am getting back a 404 error.

**Comment by arthurhsu** [comment]
Seems to be the issue of sn.im services being spotty. I can connect to it sometimes on Mac, but not always. I guess it's firewall is doing throttling and blocking users from accessing it.

**Comment by freshp86** [comment]
Can we consider dropping the shortened URL completely? I don't think it helps reduce the size in any significant way, since it is only used once at https://github.com/google/lovefield/blob/2a9e5d82a518d76949fd71d1cd70bcada2cd0865/lib/exception.js#L39. How much would the size increase if we just use
```
this.message = 'http://google.github.io/lovefield/error_lookup/src/error_lookup.html?c=' + code;
```
The benefit is that there is one less point of failure.

**Comment by krzkaczor** [comment]
Thanks for providing expanded link guys! I have just started using lovefield and it was really hard to debug it without any information about problem (shortened link doesn't work :/ )

**Comment by freshp86** [comment]
Fixed. The change will be included in next version 2.1.7.
